### PR TITLE
Add support for security manager endpoints

### LINKED ISSUE #2529: Support for organization security managers
I'm planning to add this support, using this issue to track.

- docs: [Security manager role](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization)
- docs: [API](https://docs.github.com/en/rest/orgs/security-managers)

### PR DESCRIPTION
**Pull request by jeremyhayes**
Fixes #2529

### CODE REVIEW

File: github/orgs_security_managers.go
Comment (gmlewis) [inline github/orgs_security_managers.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_security_managers.go
Comment (gmlewis) [inline github/orgs_security_managers.go:36]:
```suggestion
func (s *OrganizationsService) AddSecurityManagerTeam(ctx context.Context, org, team string) (*Response, error) {
```

File: github/orgs_security_managers.go
Comment (gmlewis) [inline github/orgs_security_managers.go:49]:
```suggestion
func (s *OrganizationsService) RemoveSecurityManagerTeam(ctx context.Context, org, team string) (*Response, error) {
```

File: github/orgs_security_managers_test.go
Comment (gmlewis) [inline github/orgs_security_managers_test.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_security_managers_test.go
Comment (gmlewis) [inline github/orgs_security_managers_test.go:51]: Please add a blank line above this.  I'm surprised `gofmt` didn't catch this.
Please make sure to run `gofmt` as described in CONTRIBUTING.md.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @jeremyhayes !
Just a few minor tweaks please.

File: github/orgs_security_managers_test.go
Comment (jeremyhayes) [reply github/orgs_security_managers_test.go:52]: Yeah, weird. Ran it multiple times to check. 🤷 

Newline added.

**Review by gmlewis — APPROVED** [review]
Thank you, @jeremyhayes !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @valbeat and @raynigon !
Merging.

**Pull request merged**
